util/util.cpp

```diff
@@ -153,90 +201,171 @@ void updateMarginTempLoop(
     {
         for (int i = 0; i < numOfZones; i++)
```

Comment:
This loop code is too long to make variable i meaningful. Recommend to use variable names like zone_id if possible.

Reply:
Agreed. It isn't my doing, though. To change it from "i" would have a ripple effect that would make this patch even larger than it is. A general refactoring should still be done, though, to get nice cleanups like this. To do it now would bloat this patch, though, and obfuscate the genuine changes I'm trying to call attention to.